From 35242738c64844c3490f99e816131eb6c373ad29 Mon Sep 17 00:00:00 2001
From: Andrew Wang <andrew.wang@cloudera.com>
Date: Wed, 20 Apr 2016 13:44:10 -0700
Subject: [PATCH 1438/2848] CLOUDERA-BUILD. Do a dist build in lib.sh.

If we do not do a dist build, not all artifacts are built.

Notably, this includes the test-sources.jar files for hadoop-tools,
meaning that those tests are not run by grind.

Change-Id: If001e43bd6baea03ec8d97fadd9a461217609507
---
 cloudera/lib.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudera/lib.sh b/cloudera/lib.sh
index a01a256..7213679 100755
--- a/cloudera/lib.sh
+++ b/cloudera/lib.sh
@@ -352,7 +352,7 @@ function setupMavenFlags() {
   local _PROTOBUF_HOME=$1
   local _TOOLCHAIN_HOME=$2
 
-  MAVEN_FLAGS="-Pnative -Drequire.fuse -Drequire.snappy -DjavaVersion=$JAVA_VERSION -DtargetJavaVersion=$TARGET_JAVA_VERSION -Dmaven.test.failure.ignore=true -Dtest.fork.count=${TEST_FORK_COUNT} -Dtest.fork.reuse=${TEST_REUSE_FORKS}"
+  MAVEN_FLAGS="-Pdist -Pnative -Drequire.fuse -Drequire.snappy -DjavaVersion=$JAVA_VERSION -DtargetJavaVersion=$TARGET_JAVA_VERSION -Dmaven.test.failure.ignore=true -Dtest.fork.count=${TEST_FORK_COUNT} -Dtest.fork.reuse=${TEST_REUSE_FORKS}"
 
   # setup of protobuf path, since Hadoop pom is using HADOOP_PROTOC_PATH it will be set here too, unless already
   # defined
-- 
1.7.9.5

